### ISSUE TITLE
Served net_version  err="the method net_version does not exist/is not available"

### ISSUE THREAD

**Issue by victorelec14** [prysmaticlabs/prysm#8124]
# 🐞 Bug Report

### Description

 I have this error in Geth after updating to version 1.0.5 (docker) and start using the Eth1 provider backup (alchemy/influra).

this error did not happen in the previous version (v1.0.4).

I already tested in geht with the http.api (old --rpcapi) with "net, eth":

>--http.api="net,eth" 

Alchemy give response:

![image](https://user-images.githubusercontent.com/543450/102260676-112ba900-3f11-11eb-961d-7e2ca052bd21.png)

### Has this worked before in a previous version?

 Yes, the previous version in which this bug was not present (v1.0.4 and olders)

## 🔥 Error

<pre><code>

WARN [12-15|19:01:19.795] Served net_version                       conn=172.17.0.1:51674 reqid=3 t="17.005µs" err="the method net_version does not exist/is not available"
WARN [12-15|19:01:20.573] Served net_version                       conn=172.17.0.1:51674 reqid=3 t="17.532µs" err="the method net_version does not exist/is not available"

</code></pre>

<pre><code>
 curl --data '{"method":"net_version","params":[],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545
{"jsonrpc":"2.0","id":1,"error":{"code":-32601,"message":"the method net_version does not exist/is not available"}}

</code></pre>
## 🌍  Your Environment

**Operating System:**

<pre>
  <code>
Docker  Version:  19.03.8

  </code>
</pre>

**What version of Prysm are you running? (Which release)**

<pre>
  <code>
v1.0.5 (Docker)
  </code>
</pre>

**Anything else relevant (validator index / public key)?**

**Comment by victorelec14** [comment]
net_version be deprecated ?

https://github.com/ethereum/EIPs/issues/611

**Comment by victorelec14** [comment]
Better replace **net_version** for **net_listening** ?

```
 curl --data '{"method":"net_listening","params":[],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545
{"jsonrpc":"2.0","id":1,"result":true}
```

**Comment by jbcebe** [comment]
I have the same problem

**Comment by jbcebe** [comment]
Must be related to this: https://github.com/ethereum/go-ethereum/issues/22034#issuecomment-747981775

**Comment by jbcebe** [comment]
Also affected: https://github.com/nucypher/nucypher/issues/2484

**Comment by nisdas** [comment]
Hey @jaybuidl @victorelec14 , this reason for the error has been described in #8170. The team is mostly off for the rest of the year but we will definitely take a look after.

**Comment by 0xKiwi** [comment]
This issue should be solved on the geth side, please refer to #8170 for future discussion as we are blocked by the spec currently.
